staker/validatorwallet/noop.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package validatorwallet
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/offchainlabs/nitro/arbnode/dataposter"
+	"github.com/offchainlabs/nitro/arbutil"
+	"github.com/offchainlabs/nitro/staker/txbuilder"
+)
+
+// NoOp validator wallet is used for watchtower mode.
+type NoOp struct{}
+
+func NewNoOp() *NoOp {
+	return &NoOp{}
+}
+
+func (*NoOp) Initialize(context.Context) error { return nil }
+
+func (*NoOp) Address() *common.Address { return nil }
+
+func (*NoOp) AddressOrZero() common.Address { return common.Address{} }
+
+func (*NoOp) TxSenderAddress() *common.Address { return nil }
+
+func (*NoOp) From() common.Address { return common.Address{} }
+
+func (*NoOp) ExecuteTransactions(context.Context, *txbuilder.Builder, common.Address) (*types.Transaction, error) {
+	return nil, nil
+}
+
+func (*NoOp) TimeoutChallenges(ctx context.Context, challenges []uint64) (*types.Transaction, error) {
+	return nil, nil
+}
+
+func (*NoOp) L1Client() arbutil.L1Interface { return nil }
```

Comment:
This returning `nil` is a bit dangerous because the `ValidatorTxBuilder` assumes it won't be nil. We should either add safety checks there, or it might be a bit safer if we just passed in an L1Interface to NewNoOp and stored it there.

Reply:
Done.